DxDiagram improve Expr descriptions (#620)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/edges/fromLineEndExpr.md

The current edge's data object.

---

> The specified field or expression must return `none`, `arrow`, `filledTriangle`, or `outlinedTriangle` value.

Reviewer: ```suggestion
The specified field or expression should return `none`, `arrow`, `filledTriangle`, or `outlinedTriangle`.
```
